test/framework/src/main/java/org/elasticsearch/test/AbstractBuilderTestCase.java

```diff
@@ -376,9 +377,9 @@ public void onRemoval(ShardId shardId, Accountable accountable) {
 
             if (registerType) {
                 mapperService.merge("_doc", new CompressedXContent(Strings.toString(PutMappingRequest.simpleMapping(
-                    STRING_FIELD_NAME, "type=text",
-                    STRING_FIELD_NAME_2, "type=keyword",
-                    STRING_ALIAS_FIELD_NAME, "type=alias,path=" + STRING_FIELD_NAME,
+                    TEXT_FIELD_NAME, "type=text",
+                    KEYWORD_FIELD_NAME, "type=keyword",
+                    TEXT_ALIAS_FIELD_NAME, "type=alias,path=" + TEXT_FIELD_NAME,
```

Comment:
This is the most relevant change, the rest should be just renames according to this part.